server/app/services/export/JsonExporter.java

```diff
@@ -79,59 +81,73 @@ private CfJsonDocumentContext buildJsonApplication(
       Application application, ProgramDefinition programDefinition) {
     ReadOnlyApplicantProgramService roApplicantProgramService =
         applicantService.getReadOnlyApplicantProgramService(application, programDefinition);
+
```

Comment:
Everything in this file is functionally the same as before.

Now, instead of exposing `public static void exportEntriesToJsonApplication(CfJsonDocumentContext jsonApplication, ImmutableMap<Path, Optional<?>> entries)` for `ProgramJsonSampler` to use, we make that private and expose `CfJsonDocumentContext buildJsonApplication(JsonExportData jsonExportData)`. 

This is much more straightforward: give me a `JsonExportData` and I'll give you a `CfJsonDocumentContext` using that data. `ProgramJsonSampler` can easily build a `JsonExportData` and rely on this method to produce the sample JSON data.